src/OmniSharp.DotNet/Models/DotNetProjectInformation.cs

```diff
@@ -9,12 +9,16 @@ public DotNetProjectInformation(ProjectContext projectContext, string configurat
         {
             this.Path = projectContext.RootProject.Path;
             this.Name = projectContext.ProjectFile.Name;
+            this.TargetFramework = new DotNetFramework(projectContext.TargetFramework);
 
             var outputPaths = projectContext.GetOutputPaths(configuration);
             this.CompilationOutputPath = outputPaths.CompilationOutputPath;
             this.CompilationOutputAssemblyFile = outputPaths.CompilationFiles.Assembly;
             this.CompilationOutputPdbFile = outputPaths.CompilationFiles.PdbPath;
 
+            var compilationOptions = projectContext.ProjectFile.GetCompilerOptions(targetFramework: projectContext.TargetFramework, configurationName: configuration);
```

Comment:
looks like it's gonna include the warning suppressions CLI is trying to suppress by default, so you wouldn't want them showing up in language services I guess